### PR TITLE
updating docs for compiling

### PR DESCRIPTION
**Pull request by joshfischer1108**
This may be a bit strong of an opinion for the documentation.  But  Debian10 is the only current working docker solution we have.  I'd like to focus on it during incubation so we can start releasing convenience binaries with future releases.  Please review the wording and make sure it makes sense and that you agree with it.

cc @thinker0

### CODE REVIEW

**Comment by nwangtw** [comment]
> This may be a bit strong of an opinion for the documentation. But Debian10 is the only current working docker solution we have. I'd like to focus on it during incubation so we can start releasing convenience binaries with future releases. Please review the wording and make sure it makes sense and that you agree with it.
> 
> cc @thinker0

I believe ubuntu work, but it is ok if decide that debian 10 is our main platform.

**Comment by joshfischer1108** [comment]
@nwangtw I was under the impression ubuntu18.04 was not working.  I'm fine with either container.  I think we should pick one as the official released convenience binary.  It's too much to maintain multiple distros well.  I'll change the wording around before merging.

File: website2/docs/compiling-docker.md [hunk 1]
```diff
@@ -130,8 +140,8 @@ After the commands, a new docker container is started with all the libraries and
 installed. The operation system is Ubuntu 18.04 by default. Now you can build Heron
```
Comment (nicknezis) [inline website2/docs/compiling-docker.md]: Should this line mention Debian instead of Ubuntu?

Comment (joshfischer1108) [reply website2/docs/compiling-docker.md]: Thanks for catching.  I reverted the script back ubuntu 18.04

**Comment by joshfischer1108** [comment]
I've updated the documentation a bit.  Please give another review before we merge.

**Comment by joshfischer1108** [comment]
If there are no concerns with the changes I'll merge the changes after 24 hours passes.

**Pull request merged**